src/remote_connector_dao/GraphWidget2D.py

```diff
@@ -0,0 +1,68 @@
+import pyqtgraph as pg
+from remote_connector_dao.app_styles import (
+    colors,
+    graph_line_style,
+    axis_label_style,
+    graph_title_style,
+)
+
+LINE_COLORS = [
+    colors["red_primary"],
+    colors["blue_primary"],
+    colors["green_primary"],
+    colors["black"],
+]
+
+LINE_SYMBOLS = ["s", "o", "t", "+"]
```

Comment:
Podemos usar um `dict` para isso?

```suggestion
LINES_MAP = {
  "s": colors["red_primary"],
  "o": colors["blue_primary"],
  "t": colors["green_primary"],
  "+": colors["black"],
}
```

Reply:
Caso a ordem seja importante, podemos considerar algo do tipo:

```suggestion
LINES = [
    { color: colors["red_primary"], symbol: "s" },
    { color: colors["blue_primary"], symbol: "o" },
    { color: colors["green_primary"], symbol: "t" },
    { color: colors["black"], symbol: "+" },
]
```

Dessa forma poderiamos usar
```py
for line in LINES:

# or
line = LINES[i]

```